<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDialog</class>
 <widget class="QDialog" name="MessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/newPrefix/icons/48x48/qommunicate.png</normaloff>:/newPrefix/icons/48x48/qommunicate.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QTextBrowser" name="messageEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item row="0" column="0" rowspan="2">
        <widget class="QLineEdit" name="messageInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Type message here</string>
         </property>
         <property name="maxLength">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QPushButton" name="sendButton">
         <property name="toolTip">
          <string>Send the message</string>
         </property>
         <property name="text">
          <string>&amp;Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="attachButton">
         <property name="text">
          <string>&amp;Attach...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/newPrefix/icons/document-open.png</normaloff>:/newPrefix/icons/document-open.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="notifyReadCB">
         <property name="toolTip">
          <string>Notify me when user reads the message</string>
         </property>
         <property name="text">
          <string>&amp;Notify Read</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionFiles">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/newPrefix/icons/document-multiple.png</normaloff>:/newPrefix/icons/document-multiple.png</iconset>
   </property>
   <property name="text">
    <string>F&amp;iles</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/newPrefix/icons/folder-documents.png</normaloff>:/newPrefix/icons/folder-documents.png</iconset>
   </property>
   <property name="text">
    <string>F&amp;older</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>messageInput</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>messageEdit</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
